examples/open-ai-chat-plugin/README.md

```diff
@@ -0,0 +1,77 @@
+# Welcome to your Eventual Project
+
+## Project Structure
+
+The following folder structure will be generated.
+
+```bash
+├──infra # an AWS CDK application that deploys the repo's infrastructure
+├──packages
+    ├──service # the NPM package containing the my-service business logic
+```
+
+### `infra`
+
+This is where you control what infrastructure is deployed with your Service, for example adding DynamoDB Tables, SQS Queues, or other stateful Resources.
+
+### `packages/service`
+
+This is where you add business logic such as APIs, Event handlers, Workflows and Tasks.
+
+## Deployed Infrastructure
+
+After deploying to AWS, you'll have a single stack named `open-ai-plugin` containing your Service. Take a look at the structure using the Resources view in CloudFormation. Here, you can find a list of all the Lambda Functions and other Resources that come with a Service.
```

Comment:
Can we call it a chat gpt plug-in? It's not an open ai plug-in.

Reply:
Not according to their docs. 

They call them chat plugins or openai plugins 

> OpenAI plugins connect ChatGPT to third-party applications.

https://platform.openai.com/docs/plugins/introduction

Reply:
Looks like they have not decided what to call them. Ha... The format is usable for multiple purposes, chat is just one of them. This example could be called chatgpt plugin though it's not quite chat gpt when used with lang chain.